index/secondary_index_manager.cc

```diff
@@ -144,7 +145,15 @@ view_ptr secondary_index_manager::create_view_for_index(const index_metadata& im
         builder.with_column(index_target->name(), index_target->type, column_kind::partition_key);
         // Additional token column is added to ensure token order on secondary index queries
         bytes token_column_name = get_available_token_column_name(*schema);
-        builder.with_computed_column(token_column_name, bytes_type, column_kind::clustering_key, std::make_unique<token_column_computation>());
+        column_computation_ptr computation;
+        if (service::get_local_storage_service().db().local().features().cluster_supports_correct_idx_token_in_secondary_index()) {
```

Comment:
GLOBAL ALERT!!!
Maybe store a reference to storage_service in secondary_index_manager as a field?

Reply:
Probably not to be addressed in this PR though. It would make it way bigger than necessary.